docker-compose-azure.yml

```diff
@@ -0,0 +1,19 @@
+version: "3"
```

Comment:
You know, now that Im looking at it, I'm not sure there are even any real differences between this and the original compose file, other than the volume.

And technically, if you build just the container, volume mapping shouldn't be needed since we already copy everything required into the container. The volume mapping is only for if you want to persist to your host, or if you want to make changes on the host (developing) and have the reflected in the running container.

Reply:
@mtheoryx , as you mentioned, the main difference is that docker-compose-azure.yml will pull from the noopkat/electric-io repo, set the platform to Azure, and use a volume to persist data on the host.  This allows us a very easy mechanism to deploy to Azure and make modifications by reapplying the docker-compose file in the Azure Web App container settings.  It will also support our proposed workshop by allowing attendees to deploy, make changes, then lock their dashboard down and still have their previous changes persist.  

While the changes are subtle, the intent in each docker-compose file is different i.e. one supports deployment to Azure out of box, the other supports local dev.